src/common/address.h

```diff
@@ -206,13 +206,22 @@ const char * fmt_addr32(uint32_t addr);
 
 MOCK_DECL(int,get_interface_address6,(int severity, sa_family_t family,
 tor_addr_t *addr));
+MOCK_DECL(int,get_loopback_address6,(int severity, sa_family_t family,
+tor_addr_t *addr));
 void interface_address6_list_free_(smartlist_t * addrs);// XXXX
+
+void free_interface_address6_list(smartlist_t * addrs);
```

Comment:
Where is this function defined?
Why can't you use interface_address6_list_free()?

Reply:
Turns out I use `interface_address6_list_free()` everywhere. Removed this in b8645a82ab0bbc7a8adb879024c1ea2e31f5f270.